week2/2-website/css/style.css

```diff
@@ -15,5 +42,199 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+img {
+  max-width: 100%;
+}
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  border-bottom: 1px solid #ddd;
+  width: 100%;
+}
+.logo {
+  width: 2.5rem;
+  margin-left: 15rem;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-around;
+  margin-bottom: 0.5rem;
+  margin-right: 5rem;
+}
+
+.navbar ul {
+  display: flex;
+  list-style: none;
+}
+
+.navbar li {
+  padding: 1rem;
+  color: #838994;
+  font-size: 1.7rem;
+  letter-spacing: 0.1rem;
+  text-decoration: none;
+}
+.menu li a:hover {
+ color: #333333;
+}
+li a.active {
+  color: #4c5058;
+  font-weight: 500;
+}
+.card img {
+  width: 12.7rem;
```

Comment:
Instead of using an absolute value like rem. You could have used `width: 33%;` on the card (not the image), to keep all the cards the same width.